src/components/PopularCoursesLayout/PopularCoursesContainer/PopularCard/PopularCard.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { StyledLinkMore, StyledPopularCardDescription, StyledPopularCardHeader, StyledPopularCardImage, StyledPopularTags, StyledYellowButton } from './styledPopularCard'
+import { StyledFlex } from '../../../../styledConstants'
+import { StyledPopularHorizontalContainer } from '../StyledPopularCourses'
+import { NavLink } from 'react-router-dom'
+import ArrowRight from '../../../../assets/icons/ArrowRight'
+
+interface IPopular {
+  header: string,
+  type: "Design" | "Data Science" | "Programming",
+  countLessons: number,
+  description: string,
+  imageSrc: string,
+  isVertical?: boolean
+}
+const PopularCard = ({header, type, countLessons, description, imageSrc, isVertical}: IPopular) => {
+  return (
+    <StyledFlex>
+      <StyledPopularHorizontalContainer>
+        <StyledPopularCardHeader>{header}</StyledPopularCardHeader>
+        <StyledPopularTags $type={type} $vertical={isVertical}><span>{type}</span><span>Beginner friendly</span><span>{countLessons} lessons</span></StyledPopularTags>
+        <StyledPopularCardDescription $vertical={isVertical}>{description}</StyledPopularCardDescription>
+        <div>
+          <StyledYellowButton>Let's start</StyledYellowButton>
+          <StyledLinkMore to="#">Learn more <ArrowRight/></StyledLinkMore>
+        </div>
+      </StyledPopularHorizontalContainer>
+      <StyledPopularCardImage $vertical={isVertical} src={imageSrc} alt="img" />
```

Comment:
You should wrap image in wrapper, so it can be more flexible in adaptives. Give it a fixed height and width, and img should be 100% of wrapper width and height.